packages/renderer/src/lib/ui/DetailsPage.svelte

```diff
@@ -50,7 +50,7 @@ export function close(): void {
     <div class="flex flex-row px-2 border-b border-charcoal-400">
       <slot name="tabs" />
     </div>
-    <div class="h-full bg-charcoal-900">
+    <div class="h-full bg-charcoal-900" style="min-height:0px">
```

Comment:
Use the tailwind option there ?

```suggestion
    <div class="h-full bg-charcoal-900 min-h-0">
```